src/util/docCookies.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Helper module use to get, set and remove item from cookie
+ *
+ * See more:
+ *  https://developer.mozilla.org/en-US/docs/Web/API/document.cookie
+ *  https://developer.mozilla.org/User:fusionchess
+ *  https://github.com/madmurphy/cookies.js
+ *  This framework is released under the GNU Public License, version 3 or later.
+ *  http://www.gnu.org/licenses/gpl-3.0-standalone.html
+ */
+
+/**
+ * Get specific key's value stored in cookie
+ *
+ * @param {string} sKey
+ *
+ * @returns {string|null}
+ */
+export function getItem(sKey) {
+  if (!sKey) return null;
+
+  return decodeURIComponent(
+    // eslint-disable-next-line no-useless-escape
+    document.cookie.replace(new RegExp('(?:(?:^|.*;)\\s*' + encodeURIComponent(sKey).replace(/[\-\.\+\*]/g, '\\$&') + '\\s*\\=\\s*([^;]*).*$)|^.*$'), '$1')) || null;
+}
+
+/**
+ * Set specific key's value in cookie
+ *
+ * @param {string} sKey cookie name
+ * @param {string} sValue cookie value
+ * @param {string} vEnd expire|max-age
+ * @param {string} sPath path of current item
+ * @param {string} sDomain domain name
+ * @param {boolean} bSecure true|false
+ *
+ * @returns {true}
+ */
+export function setItem(sKey, sValue, vEnd, sPath, sDomain, bSecure) {
+  let sExpires = "";
+  if (vEnd) sExpires = `; expires=` + vEnd.toUTCString();
+
+  document.cookie = encodeURIComponent(sKey) + '=' + encodeURIComponent(sValue) + sExpires + (sDomain ? `; domain=` + sDomain : '') + (sPath ? `; path=` + sPath : '') + (bSecure ? `; secure` : '');
+
+  return true;
+}
+
+/**
+ * Remove specific key's value from cookie
+ *
```

Comment:
🤦‍♂ I just realized `hasItem` isn't even implemented! :( Let's not worry about fixing it now, but let's mark it so it's obvious.

```suggestion
 * BROKEN Remove specific key's value from cookie
 * @fixme hasItem isn't even implemented! This will always error.
```